crates/oxvg_style/src/lib.rs

```diff
@@ -396,22 +395,38 @@ impl TryFrom<&Attribute> for SVGStyle {
                 )
             });
             let v = LIST_SEP_SPACE.replace_all(&v, "$a, ");
-            value = ROTATE
-                .replace_all(&v, |caps: &regex::Captures| {
-                    format!("{}({}deg", &caps["f"], &caps["v"])
-                })
-                .to_string();
+            let v = LIST_SEP_FIX.replace_all(&v, ")");
+            let v = ROTATE.replace_all(&v, |caps: &regex::Captures| {
+                format!("{}({}deg", &caps["f"], &caps["v"])
+            });
+            let v = FUNC_SEP.replace_all(&v, ") ");
+            log::debug!("tried making transform compatible: {value} -> {v}");
+            value = v.to_string();
```

Comment:
We're doing too much here to get `transform` attributes working. Let's instead create a parser in the same vein as lightning css for presentation attributes.

We may want to fork our work into lightningcss when ready